kolibri/plugins/coach/assets/src/views/lessons/LessonResourceSelectionPage/index.vue

```diff
@@ -0,0 +1,244 @@
+<template>
+
+  <div class="selection-page">
```

Comment:
:flags: new `immersive-page` component to be derived from this page. Hopefully killing off `immersive-fullscreen-modal` afterwards